cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -916,9 +871,16 @@ impl ABIBody for AArch64ABIBody {
             }
         }
 
+        // N.B.: "nominal SP", which we use to refer to stackslots
+        // and spillslots, is *here* (the value of SP at this program point).
+        // If we push any clobbers below, we emit a virtual-SP adjustment
+        // meta-instruction so that the nominal-SP references behave as if SP
+        // were still at this point.
```

Comment:
As said below, I think this comment should be placed closer to `MemArg::NominalSPOffset`, with a drawing of what it should look like.